index.js

```diff
@@ -22,25 +22,28 @@ const helmet = {
 const middlewares = Object.keys(helmet)
 
 module.exports = fp(function (fastify, opts, next) {
-  middlewares.forEach(function (middlewareName) {
-    const middleware = helmet[middlewareName]
-    const option = opts[middlewareName]
-    const isDefault = config.defaultMiddleware.indexOf(middlewareName) !== -1
+  fastify
+    .register(require('middie'))
+    .after(() => {
+      middlewares.forEach(function (middlewareName) {
+        const middleware = helmet[middlewareName]
+        const option = opts[middlewareName]
+        const isDefault = config.defaultMiddleware.indexOf(middlewareName) !== -1
 
-    if (option === false) { return }
+        if (option === false) { return }
 
-    if (option != null) {
-      if (option === true) {
-        fastify.use(middleware({}))
-      } else {
-        fastify.use(middleware(option))
-      }
-    } else if (isDefault) {
-      fastify.use(middleware({}))
-    }
-  })
-
-  next()
+        if (option != null) {
+          if (option === true) {
+            fastify.use(middleware({}))
+          } else {
+            fastify.use(middleware(option))
+          }
+        } else if (isDefault) {
+          fastify.use(middleware({}))
+        }
+      })
+      next()
+    })
 }, {
   fastify: '>=0.39.1',
```

Comment:
This needs to be bumped.

Reply:
It would fail the build and i want the tests passing
I'll add it as a todo